crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -615,6 +616,18 @@ impl_from!(i32, i128);
 impl_from!(i64, i128);
 impl_from!(isize, i128);
 
+impl FromStr for Felt {
+    type Err = FromStrError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
can you add rust doc over it to say that it accepts decimals or hex prefixed by 0x plz

Reply:
added in 2fc06e4789b005624665d1ce69a6afe278b3673a